pkg/containers/containers.go

```diff
@@ -373,6 +373,19 @@ func (c *Containers) GetCgroupInfo(cgroupId uint64) CgroupInfo {
 	return cgroupInfo
 }
 
+// GetContainers provides a list of all existing containers.
+func (c *Containers) GetContainers() []CgroupInfo {
+	var conts []CgroupInfo
+	c.mtx.RLock()
+	defer c.mtx.RUnlock()
+	for _, v := range c.cgroups {
+		if v.ContainerId != "" && v.expiresAt.IsZero() {
```

Comment:
Wer'e mapping cgroups to container ids.
Having a cgroup doesn't mean that this is what we refer to as a "container".
For example, systemd uses cgroups for other stuff.
In these cases, where the cgroup path doesn't container a valid container id (64 chars long hex string), it will just be empty.